app/config.go

```diff
@@ -485,50 +519,59 @@ func NewHostedChains() *types.HostedBlockchains {
 		// if does not exist create one
 		jsonFile, err = os.OpenFile(chainsPath, os.O_RDWR|os.O_CREATE, os.ModePerm)
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
 		}
 		// generate hosted chains from user input
 		c := GenerateHostedChains()
 		// create dummy input for the file
 		res, err := json.MarshalIndent(c, "", "  ")
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
 		}
 		// write to the file
 		_, err = jsonFile.Write(res)
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
 		}
 		// close the file
 		err = jsonFile.Close()
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
 		}
 	}
 	// reopen the file to read into the variable
 	jsonFile, err := os.OpenFile(chainsPath, os.O_RDWR|os.O_CREATE, os.ModePerm)
 	if err != nil {
-		panic(NewInvalidChainsError(err))
+		fmt.Println(NewInvalidChainsError(err))
+		os.Exit(1)
 	}
 	bz, err = ioutil.ReadAll(jsonFile)
 	if err != nil {
-		panic(NewInvalidChainsError(err))
+		fmt.Println(NewInvalidChainsError(err))
+		os.Exit(1)
 	}
 	// unmarshal into the structure
 	var hostedChainsSlice []types.HostedBlockchain
 	err = json.Unmarshal(bz, &hostedChainsSlice)
 	if err != nil {
-		panic(NewInvalidChainsError(err))
+		fmt.Println(NewInvalidChainsError(err))
+		os.Exit(1)
 	}
 	// close the file
 	err = jsonFile.Close()
 	if err != nil {
-		panic(NewInvalidChainsError(err))
+		fmt.Println(NewInvalidChainsError(err))
+		os.Exit(1)
 	}
 	m := make(map[string]types.HostedBlockchain)
 	for _, chain := range hostedChainsSlice {
 		if err := nodesTypes.ValidateNetworkIdentifier(chain.ID); err != nil {
 			panic(fmt.Sprintf("invalid ID: %s in network identifier in %s file", chain.ID, GlobalConfig.PocketConfig.ChainsName))
+			os.Exit(1)
```

Comment:
Rogue panic!!

```suggestion
                      log.Fatalf("invalid ID: %s in network identifier in %s file", chain.ID, GlobalConfig.PocketConfig.ChainsName)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work